### ISSUE TITLE
[Skia] Add Splash screen support

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#10822]
## For which Platform:

- [ ] WebAssembly
- [ ] Android
- [ ] iOS
- [ ] macOS (AppKit)
- [ ] Mac Catalyst
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Linux Framebuffer
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by MartinZikmund** [comment]
When implemented, ensure issues like https://github.com/unoplatform/uno/issues/10178 are avoided (app content should render only after `Window.Activate()`):

```
await Task.Delay(5000);
// Here splash screen should still be displayed!

Window.Activate();
// Now splash screen should dismiss
```

**Comment by MartinZikmund** [comment]
We now provide `ExtendedSplashScreen` in Toolkit, and Desktop targets launch very quickly, so this feature is no longer needed
